app/uk/gov/hmrc/tai/viewModels/TaxCodeDescriptor.scala

```diff
@@ -29,80 +30,82 @@ trait TaxCodeDescriptor {
 
   case class TaxCodeDescription(taxCode: String, basisOperation: BasisOperation, scottishTaxRateBands: Map[String, BigDecimal])
 
-  def describeTaxCode(taxCode: String, basisOperation: BasisOperation, scottishTaxRateBands: Map[String, BigDecimal])
+  def describeTaxCode(taxCode: String, basisOperation: BasisOperation, scottishTaxRateBands: Map[String, BigDecimal], currentTaxCode: Boolean = true)
```

Comment:
Could 'currentTaxCode' be named with an 'is' before to make it clearer that it's a boolean rather than an actual tax code?

Reply:
Agreed, it's done now